lib/engine/game/g_1849/corporation.rb

```diff
@@ -5,7 +5,7 @@ module Game
     module G1849
       class Corporation < Engine::Corporation
         attr_reader :token_fee
-        attr_accessor :next_to_par, :closed_recently, :slot_open, :reached_max_value, :sms_hexes
+        attr_accessor :next_to_par, :closed_recently, :slot_open, :reached_max_value, :sms_hexes, :e_token
```

Comment:
Should e_token be an ability? That way other players can see the corporation has it.

Reply:
I enabled the display on corp charters with this code in game.rb: 

https://github.com/philcampeau/18xx/blob/b5487960d95197a18c153d5245c9d3ac97a47775/lib/engine/game/g_1849/game.rb#L939

```     
def status_array(corporation)
   return unless corporation.e_token
   ['E-Token Purchased']
end
```

Reply:
Last I remember from slack, you decided to make `e_token` an ability, so this accessor shouldn't be needed.